### ISSUE TITLE
daterangepicker on ajax loaded content

### ISSUE THREAD

**Issue by maczj** [dangrossman/bootstrap-daterangepicker#1326]
I have stucked with this problem. The datepicker is working fine on static content.
But fails when content is loaded via Ajax. Please help me

**Comment by maczj** [comment]
i have found a way..

**Comment by maczj** [comment]
finally i can load the daterange picker on ajax content, but somehow it won't closed when the range date has been selected. here is some of my code

 	<script>
	function initdatepicker(){
			$('#demo').daterangepicker({
				  locale: {
				  format: 'DD-MM-YYYY',
				  "daysOfWeek": [
						"Mg",
						"Sn",
						"Sl",
						"Rb",
						"Km",
						"Jm",
						"Sb"
					],
					"monthNames": [
						"Januari",
						"Februari",
						"Maret",
						"April",
						"Mei",
						"Juni",
						"Juli",
						"Augustus",
						"September",
						"Oktober",
						"November",
						"Desember"
					],
				  },
					"singleDatePicker": true,
					"showDropdowns": true,
				}, function(start, end, label) {
				  console.log("New date range selected: ' + start.format('YYYY-MM-DD') + ' to ' + end.format('YYYY-MM-DD') + ' (predefined range: ' + label + ')");
		});	
	}
	
	function statusBeli(statusBeli, div)
	{
		$.ajax({
			type:"GET",
			url:"form_date.php",    
			data: "status=" + status,
			success: function(data){    
				$("#"+div).html(data);
				initdatepicker();             
			},
			error: function(){
				alert("error");
			}
		});
	}
	$(document).ready(function(){
		    initdatepicker();
	});
</script>

        <select name="status" onChange="statusBeli(this.value,'lamautang');"  class="form-control">
        <option value="-">Pilih Metode Pembelian</option>
        <option value="0">PLAIN INPUT TEXT</option>
        <option value="1">DATE PICKER</option>

        </select>

and here is my form_date.php

	if($status_beli==0)
	{
?>
<input type="text" id="demo" name="demo" readonly="readonly">
<?
        }
        else if($status_beli==1)
       {
       ?>
        <div class="demo">
      		<input type="text" id="demo" name="demo" readonly="readonly">
			<i class="glyphicon glyphicon-calendar fa fa-calendar"></i>
		</div>
          <?php
           } ?>

**Comment by maczj** [comment]
![untitled](https://cloud.githubusercontent.com/assets/19758620/21482470/69f8c506-cbaf-11e6-9f47-0363d527d406.png)

the date picker window won't closed even i have choosen the date
